docs/install_linux_mac.rst

```diff
@@ -265,7 +266,7 @@ Now install the pre-requirements with apt:
 
 .. code-block:: none
 
-    sudo apt -y install python3.8 python3.8-dev python3.8-venv python3-pip git openjdk-11-jre-headless build-essential
+    sudo apt -y install python3.8 python3.8-dev nano python3.8-venv python3-pip git openjdk-11-jre-headless build-essential
```

Comment:
The requirements are grouped for easier maintenance so it would be better to not put `nano` in-between python requirements:
```suggestion
    sudo apt -y install python3.8 python3.8-dev python3.8-venv python3-pip git openjdk-11-jre-headless build-essential nano
```